models/MessagesModel.js

```diff
@@ -38,7 +38,15 @@ const getConversationList = `
 const getOtherUser = `
   SELECT
     username,
-    "displayName"
+    "displayName",
+    "bio",
+    "joinedDate",
+    (
+      SELECT
+        COUNT(*)
+      FROM follow
+      WHERE "followedUserId" = "userId"
+    ) AS "followerCount"
   FROM app_user
```

Comment:
I got a little confused where "userId" came from. Can you alias the table and use that in the query?

```suggestion
      WHERE "followedUserId" = u."userId"
    ) AS "followerCount"
  FROM app_user AS u
```